pegasus/sites.v3/code.org/views/page_not_available.haml

```diff
@@ -0,0 +1,15 @@
+:ruby
+  @header["title"] = I18n.t(:page_not_available_title)
+
+%link{href: "/css/errors.css", rel: "stylesheet"}
+.error-parent
+  .error-child
+    %img{src: '/shared/images/sad-bee-avatar.png'}
+    %br
+    .error-message
+      %h1= I18n.t(:page_not_available_title)
+      = I18n.t(:page_not_available_description).html_safe
```

Comment:
@Hamms is this an example where we shouldn't insert HTML straight from a translator? [Here's a link to a Slack discussion where we discussed this in more detail.](https://codedotorg.slack.com/archives/C0T0UQH0R/p1594846525154300)

Reply:
Hmm Hamms is out this week, here's an example he helped me with:

https://github.com/code-dot-org/code-dot-org/blob/eb7a9f474743e2b2d84afe8bf3a14c5ae1ba11f9/dashboard/app/views/layouts/_thank_donors_interstitial.html.haml#L19

There's more context in the slack thread, and I just posted in i18n to see if anyone has any more information to share.

Reply:
updated to use markdown instead!